draft-thomson-webpush-protocol.xml

```diff
@@ -749,6 +680,8 @@ DATA         [stream 4] +END_STREAM
          However, the amount of information that is exposed can be limited.
        </t>
        <t>
+         <!-- Is "Subscription URIs" adequate in this context? Is it clear that
+              this means "monitor" and "push"? -->
```

Comment:
I believe that this section relates to "push" URIs only.  The split means that "monitor" URIs can't be correlated anyway.